### ISSUE TITLE
callBack method to know if a PDF has generated or not

### ISSUE THREAD

**Issue by rob-ot-dot-be** [mpdf/mpdf#161]
### I have this problem/would like to have this functionality
Is there a callback method I can address to know if a PDF is generated or has failed ?

**Comment by finwe** [comment]
No, there is not ATM. Leaving this issue here as an enhancement proposal
